ui/package.json

```diff
@@ -13,6 +14,9 @@
     "@types/react": "^18.0.28",
     "@types/react-dom": "^18.0.11",
     "antd": "^5.2.1",
+    "axios": "^1.3.4",
```

Comment:
we can remove these 3 packages if not used.

Reply:
fixed, remove mockjs and axios.